webapp/src/components/AssetCard/AssetCard.tsx

```diff
@@ -35,19 +43,65 @@ const RentalPrice = ({
   )
 }
 
+const RentalChip = ({
+  asset,
+  rental,
+  isClaimingBackLandTransactionPending
+}: {
+  asset: Asset
+  isClaimingBackLandTransactionPending: boolean
+  rental: RentalListing | null
+}) => {
+  const rentalEndDate: Date | null = useMemo(
+    () => (rental ? getRentalEndDate(rental) : null),
+    [rental]
+  )
+  const rentalHasEnded = rental ? hasRentalEnded(rental) : false
+
+  return (
+    <div className="LandBubble">
+      {isClaimingBackLandTransactionPending ? (
+        <>
+          <Icon className="warning-icon" name="warning sign" />{' '}
+          {t('asset_card.rental_bubble.claiming_back', {
+            asset: asset.category
+          })}
+        </>
+      ) : isRentalListingOpen(rental) ? (
+        t('asset_card.rental_bubble.listed_for_rent')
+      ) : isRentalListingExecuted(rental) && !rentalHasEnded ? (
+        t('asset_card.rental_bubble.rented_until', { endDate: rentalEndDate })
+      ) : isRentalListingExecuted(rental) && rentalHasEnded ? (
+        <>
+          <Icon className="warning-icon" name="warning sign" />
+          {t('asset_card.rental_bubble.rental_ended')}
+        </>
+      ) : null}
+    </div>
+  )
+}
+
 const AssetCard = (props: Props) => {
   const {
     asset,
     isManager,
     price,
-    rentalPricePerDay,
     showListedTag,
+    showRentalBubble,
     onClick,
-    isClaimingBackLandTransactionPending
+    isClaimingBackLandTransactionPending,
+    rental
   } = props
 
   const title = getAssetName(asset)
   const { parcel, estate, wearable, emote, ens } = asset.data
+  const rentalPricePerDay: string | null = useMemo(
+    () =>
+      rental && rental.status === RentalStatus.OPEN
```

Comment:
```suggestion
      isRentalListingOpen(rental)
```
we could use the util here

Reply:
Changing it!